diff --git a/pkg/proc/target.go b/pkg/proc/target.go
index 8012394..9012093 100644
--- a/pkg/proc/target.go
+++ b/pkg/proc/target.go
@@ -363,7 +363,7 @@ func setAsyncPreemptOff(p *Target, v int64) {
 		return
 	}
 	if debugv.Unreadable != nil {
-		logger.Warnf("runtime/debug variable unreadable: %v", err, debugv.Unreadable)
+		logger.Warn("runtime/debug variable unreadable: ", err, debugv.Unreadable)
 		return
 	}
 	asyncpreemptoffv, err := debugv.structField("asyncpreemptoff") // +rtype int32
